analysis/import/MetricGardenerImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/metricgardenerimporter/AttributeDescriptors.kt

```diff
@@ -6,11 +6,11 @@ fun getAttributeDescriptors(): Map<String, AttributeDescriptor> {
     val descriptors = mutableMapOf<String, AttributeDescriptor>()
     val npmLink = "https://www.npmjs.com/package/metric-gardener"
 
-    descriptors["mcc"] = AttributeDescriptor(title = "Maximum Cyclic Complexity", description = "Maximum cyclic complexity", link = npmLink)
+    descriptors["mcc"] = AttributeDescriptor(title = "Maximum Cyclic Complexity", description = "Maximum cyclic complexity based on paths through the code by McCabe", link = npmLink)
     descriptors["functions"] = AttributeDescriptor(title = "Number of Functions", description = "Number of functions", link = npmLink)
     descriptors["classes"] = AttributeDescriptor(title = "Number of Classes", description = "Number of classes", link = npmLink)
     descriptors["lines_of_code"] = AttributeDescriptor(title = "Lines of Code", description = "Lines of code including empty lines and comments", link = npmLink)
-    descriptors["comment_lines"] = AttributeDescriptor(title = "Comment Lines", description = "Number of lines containing either comment or commented-out code", link = npmLink)
+    descriptors["comment_lines"] = AttributeDescriptor(title = "Comment Lines", description = "Number of lines containing either a comment or commented-out code", link = npmLink)
```

Comment:
In metricTitle.js: "Number of Code Lines with Comments"

Reply:
I prefer the current title, because it is shorter and still understandable.